scripts/upload-artifacts.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -ev
+export ARTIFACTS_BUCKET="jpt-travis-ci-pipeline"
```

Comment:
"JPT Travis CI pipeline" has a teeny-tiny scent of Bitbucket Pipelines
```suggestion
export ARTIFACTS_BUCKET="jpt-travis-ci-artifacts"
```
Just a nit 😉